src/pudl/transform/ferc1.py

```diff
@@ -53,15 +46,14 @@ class Ferc1Source(enum.Enum):
 
 @enum.unique
 class Ferc1TableId(enum.Enum):
-    """Enumeration of the allowable FERC 1 table IDs.
+    """Enumeration of the allowed FERC 1 table IDs.
 
-    Hard coding this seems bad. Somehow it should be either defined in the context of
-    the Package, the Ferc1Settings, an etl_group, or DataSource. All of the table
-    transformers associated with a given data source should have a table_id that's
+    Hard coding this doesn't seem ideal. Somehow it should be either defined in the
+    context of the Package, the Ferc1Settings, an etl_group, or DataSource. All of the
+    table transformers associated with a given data source should have a table_id that's
```

Comment:
agreed! I think defining this in the `Ferc1Settings` seems correct to me on the face of it. but this isn't a terrible thing and I agree that this works for now.

Reply:
Okay, let's revisit it later.